.github/workflows/ci.yaml

```diff
@@ -48,6 +50,26 @@ jobs:
           python -m pip install --upgrade pip wheel
           pip install -r dev-requirements.txt
           pip install -r requirements.txt
-      - name: Run API tests
+      - name: Run integration tests
         run: |
-          pytest tests/integration/api --env=qa
+          python -m pytest tests/integration --env=qa
+      - name: Run functional tests
+        run: |
+          python -m pytest tests/functional --env=qa
+
+  trigger-prod-deployment:
```

Comment:
I think this workflow is pretty well scoped to just running the tests. 

I recommend actually adding this job/step in the build-qa.yaml file. We can change it to deploy.yaml.